examples/mobile/ScheduleApp/index.html

```diff
@@ -26,6 +26,15 @@
 			<div class="ui-appbar-action-buttons-container">
 				<a class="ui-btn ui-btn-icon ui-btn-icon-add" data-style="flat" href="#turn-on-dialog"></a>
 				<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat"></button>
+				<select data-native-menu="false" id="theme-selector" class="app-hidden">
+					<option value="choose-one" data-placeholder="true">Theme</option>
+					<option value="light">
+						Light theme
```

Comment:
please be consistent in capitals for Theme
Light **t**heme
vs
Dark **T**heme

Reply:
Done!